ui/regions/RegionBuilderViewController.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  RegionBuilderViewController.swift
+//  org.onebusaway.iphone
+//
+//  Created by Aaron Brethorst on 8/24/16.
+//  Copyright © 2016 OneBusAway. All rights reserved.
+//
+
+import UIKit
+import OBAKit
+import SVProgressHUD
+
+@objc protocol RegionBuilderDelegate {
+    func regionBuilderDidCreateRegion(region: OBARegionV2)
+}
+
+class RegionBuilderViewController: OBAStaticTableViewController {
```

Comment:
A ton of the rows that are displayed here never actually get used. Hide or delete them.